src/About/About.tsx

```diff
@@ -0,0 +1,38 @@
+import './About.scss'
+import React, { useState, useEffect } from "react";
+import aqtable from '../assets/ap3.jpg'
+
+const About: React.FC = () => {
+  return (
+   <section className="about-section">
```

Comment:
This page looks AWESOME! Great idea, I think it's a wonderful addition to our page!